tests/test_aliases.py

```diff
@@ -171,17 +192,16 @@ class Child(Parent):
         y: str
         z: str
 
-    assert [f.alias for f in Parent.model_fields.values()] == ['abc', 'y']
+    assert [f.alias for f in Parent.model_fields.values()] == ['abc', None]
     assert [f.alias for f in Child.model_fields.values()] == ['abc', 'Y', 'Z']
 
 
-@pytest.mark.xfail(reason='working on V2')
 def test_low_priority_alias():
-    # TODO: alias_priority has been removed from `Field`. Should we re-add it?
-    #   Is there something new that can be used to replicate this functionality?
-    #   See discussion in https://github.com/pydantic/pydantic/pull/5181/files#r1137618854
-    #   Either way, if we don't re-add it to `Field`, remember to update the migration guide
+    # TODO:
+    #  Alternative 1: we could drop alias_priority and tell people to manually override aliases in child classes
+    #  Alternative 2: we could add a new argument `override_with_alias_generator=True` equivalent to `alias_priority=1`
     class Parent(BaseModel):
+        w: bool = Field(..., alias='w_')
         x: bool = Field(..., alias='abc', alias_priority=1)
```

Comment:
This is the only TODO I have left here. @samuelcolvin do you have any suggestions for what to do here?

If no strong opinions, I'm inclined to just keep it as is for backwards compatibility and drop the TODO comment.